src/Sylius/Behat/Context/Ui/Admin/ManagingProductVariantsContext.php

```diff
@@ -170,6 +171,14 @@ public function iSetItsPriceTo($price = null, $channel = null)
     }
 
     /**
+     * @When /^I set its original price to "(?:€|£|\$)([^"]+)" for "([^"]+)" channel$/
+     */
+    public function iSetItsOriginalPriceTo($originalPrice = null, $channel = null)
+    {
+        $this->createPage->specifyOriginalPrice($originalPrice, (null === $channel) ? $this->sharedStorage->get('channel') :$channel);
```

Comment:
1. We should definitely pass a name of channel two that function, not an object (it would require ``__toString`` method)
2. With this step definition it's not even possible to have ``$originalPrice``` and ``$channel`` as null (as it won't be matched ;))